README.md

```diff
@@ -1,2 +1,336 @@
 # optimizer
-AsyncAPI offers many different ways to reuse certain parts of the document like messages or schemas definitions or references to external files, not to even mention the traits. There is a need for a tool that can be plugged into any workflows and optimize documents that are generated from code, but not only.
+AsyncAPI offers many ways to reuse certain parts of the document like messages or schemas definitions or references to external files, not to even mention the traits. There is a need for a tool that can be plugged into any workflows and optimize documents that are generated from code, but not only.
+## Testing
+1) Clone the project
+  `git https://github.com/asyncapi/optimizer.git`
+2) Install the dependencies
+  `npm i`
+3) open the `index.ts` file and add your testing code before export.
```

Comment:
Imho it is not a good way of testing the library, to tell the user to close and edit the main source code.  
I would just remove this section and instead using examples from Usage section, create `index.js` file in `examples` directory. And create an npm script that I can call to run the example, like `asyncapi run example` and then in readme you can just have a clear information that you can quickly check the library by running the example, and that user can modify the example and put their file as an input.

Reply:
@derberg done.